src/js/components/Values/IssueImageDisplay.jsx

```diff
@@ -16,13 +16,14 @@ export default class IssueImageDisplay extends Component {
       return null;
     }
 
-    const oneIssue = IssueStore.getIssueByWeVoteId(this.props.issueWeVoteId);
 
+    const oneIssue = IssueStore.getIssueByWeVoteId(this.props.issueWeVoteId);
+//   console.log(oneIssue.issue_name)
```

Comment:
I would request having the "//" indented so it is part of the indented code block, as opposed at the start of the line file.